src/components/Facility/DischargedPatientsList.tsx

```diff
@@ -281,7 +281,9 @@ const DischargedPatientsList = ({
           <div className="flex-1">
             <CountBlock
               text="Discharged Patients"
-              count={count}
+              count={
+                facilityQuery.loading || count === undefined ? null : count
+              }
               loading={facilityQuery.loading}
```

Comment:
Can't we do this 🤔. 
```suggestion
              count={count}
              loading={facilityQuery.loading || count === undefined}
```
I am really not sure when the ui would not be in a loading state where count is`undefined`